.cirrus.yml

```diff
@@ -1,6 +1,6 @@
 container:
   image: smartrent/terraform-ci:0.14.4
-  cpu: 0.5
+  cpu: 1
```

Comment:
Unrelated changes to PR: some reason (possibly since repo is public now), 0.5 was getting an error `Requested memory is too high! You can request at most 4G per CPU`. Possibly a limitation with non-integers. 🤷‍♀️ I tried a lower value `0.25` but that had the same error.

Reply:
weird! well thats good to be aware of. Thanks for fixing that!